chapters/VK_EXT_acquire_drm_display/acquire_drm_display.txt

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='vkAcquireDrmDisplayEXT',desc='Acquire access to a VkDisplayKHR using DRM',type='protos']
+--
+
+To acquire permission to directly a display in Vulkan from the Direct Rendering
+Manager (DRM) interface, call:
+
+include::{generated}/api/protos/vkAcquireDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:drmFd Connection to the DRM resource.
+  * pname:display The display the caller wishes Vulkan to control.
+
+All permissions necessary to control the display are granted to the Vulkan
+instance associated with the provided pname:physicalDevice until the display is
+either released or the connection is closed.
+If any error is encountered during the acquisition of the display, the call
+must return the error code ename:VK_ERROR_INITIALIZATION_FAILED.
+
+include::{generated}/validity/protos/vkAcquireDrmDisplayEXT.txt[]
+--
+
+[open,refpage='vkGetDrmDisplayEXT',desc='Query the VkDisplayKHR corresponding to a DRM connector ID',type='protos']
+--
+
+Before acquiring a display from the DRM interface, the caller may want to
+select a specific sname:VkDisplayKHR handle by identifying it using a
+pname:connectorId. To do so, call:
+
+include::{generated}/api/protos/vkGetDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device to query the display from.
+  * pname:drmFd Connection to the DRM node file descriptor.
+  * pname:connectorId Identifier of the specified DRM connector.
```

Comment:
Somewhat related to above, suggest adding "on the DRM device associated with drmFd" or something to clarify this a bit. I forget what the namespace of connector IDs is, but I assume you can't correlate connectors on other DRM devices using any random DRM fd + connector ID.

Alternatively, explicit valid usage could be added to clear this up.

Reply:
It should be clarified yes. Same as above, I'm hesitating between adding a `VK_INVALID_CONNECTOR_EXT` or `VK_ERROR_UNKNOWN`.

Reply:
Vulkan doesn't usually check things like this in driver implementations. It just asserts the parameters must be valid and compatible in some way in valid usage language in the spec, and then validation layers would do any possible checking/reporting of errors. Drivers are free to crash and burn or return any random error they please (undefined behavior) if valid usage is violated.

Reply:
So I could just return `VK_ERROR_UNKNOWN` and set errno to EINVAL ?

Reply:
That'd be one valid implementation, I believe, but the spec shouldn't require any particular behavior, nor should portable apps rely on any particular behavior in this case. E.g., you can't conformance test it because the driver could conceivably segfault in this case without being non-conformant.

Reply:
I'll go with `VK_ERROR_UNKNOWN` and leave a note about the fact that the implementation could set errno to an adequate value.